scripts/helper/create-cluster.sh

```diff
@@ -276,48 +283,20 @@ case "$OCP_VERSION" in
 4.4|4.5)
 	patch -p1 < $WORKSPACE/ocs-upi-kvm/files/ocp4-upi-kvm.legacy.patch
```

Comment:
Let's make this conditional on the presence of the patch.   There is now release-4.5 branch that we can use, except it doesn't have all of the fixes that we need.  I have been advocating for fixes to be cherry picked to this release.